tests/tests_integration/test_api/test_data_modeling/test_containers.py

```diff
@@ -56,7 +57,7 @@ def test_apply_retrieve_and_delete(self, cognite_client: CogniteClient, integrat
         # Arrange
         new_container = ContainerApply(
             space=integration_test_space.space,
-            external_id="IntegrationTestContainer",
+            external_id="IntegrationTestContainer" + random_string(5),
```

Comment:
We don't want to create a new container for each test, this delete + recreate containers pattern that really only happens in tests has a bunch of undesireable effects on the system (e.g. elasticsearch mapping explosion). Let's keep it as a fixed externalId and remove the cleanup below.

Reply:
Updated now! Not much was left of the test after, so please have another look. There's also two other tests that seem to create+delete containers: `test_apply_failed_and_successful_task` and `test_load_and_create_only_required_fields`. I've left them as is - but let me know if you want them changed/removed as well 👌

Reply:
...I also... did some... comment deletion 👀 Don't tell Anders 🤭

Reply:
Yeah, let's update those aswell 👍

Reply:
Done, without the create+delete, it was already covered by other tests, so I deleted both.